src/main/java/shef/servlets/NewRecipeServlet.java

```diff
@@ -75,6 +75,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     Collection<EmbeddedEntity> tags = getParameters(request, TAG, searchStrings);
     Collection<EmbeddedEntity> ingredients = getParameters(request, INGREDIENT, searchStrings);
     Collection<EmbeddedEntity> steps = getParameters(request, STEP, null);
+    int likes = Integer.parseInt(request.getParameter("likes"));
```

Comment:
Is there a scenario where this parse could fail?

Reply:
For now, likes are getting from a number input in an HTML form, so "likes" is always an integer. Eventually, likes will probably become their own Datastore entities, so this will be removed entirely.